### PR TITLE
Lucky's misc fixes 1

### PR DESCRIPTION
**Pull request by SpaceCat404**
# Lucky's Misc Fixes 1
## About The Pull Request
This is a very small PR adressing some very minor issues. It adds sprites and item paths for the CC uniforms and adds a squirm emote.

## Why It's Good For The Game
CC Uniforms: Admins will no longer have to mess with VV to have CC stuff.
Squirm Emote: New emotes are always good for the same reason the old ones were.

## A Port?
Partially? (CC Uniforms seem to be sandstone sprites, but the item sprites are hand made)

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
add: CC Uniform item sprites
add: CC uniform item directories
add: squirm emote
/:cl:

### CODE REVIEW

File: Coder Build.bat
Comment (MosleyTheMalO) [inline Coder Build.bat:1]: This is not necessary, you can just use the run and debug option on vscode to run the server with breakpoints and stuff

**Review by MosleyTheMalO — CHANGES_REQUESTED** [review]

File: modular_splurt/code/modules/clothing/under/jobs/centcom.dm [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+/obj/item/clothing/under/rank/centcom/tactical
+	name = "tactical centcom uniform"
+	desc = "No technorussian."
+	icon_state = "military"
+	item_state = "military"
+/obj/item/clothing/under/rank/centcom/tacticalengi
+	name = "tactical engie centcom uniform"
+	desc = "\"I solve practical problems.\""
+	icon_state = "military_eng"
+	item_state = "military_eng"
```
Comment (MosleyTheMalO) [inline modular_splurt/code/modules/clothing/under/jobs/centcom.dm:53]: add spaces between the classes

File: modular_splurt/code/modules/mob/living/emotes.dm [hunk 2]
```diff
@@ -1627,3 +1627,12 @@ To add randomization to your emote, copy and paste this line of code:
...
 	volume = 30
+
+/datum/emote/sound/human/squirm
+	key = "squirm"
+	key_third_person = "squirm"
+	name = "squirm"
+	message = "squirms in place!"
+	message_mime = "squirms in place!"
+	emote_type = EMOTE_VISIBLE
+	emote_cooldown = 0.8 SECONDS // mrrp mrrp meow
```
Comment (MosleyTheMalO) [inline modular_splurt/code/modules/mob/living/emotes.dm:1638]:
```suggestion
	emote_cooldown = 0.8 SECONDS
```

**Review by MosleyTheMalO — CHANGES_REQUESTED** [review]

File: modular_splurt/code/modules/mob/living/emotes.dm (hunk 2 above)
Comment (SpaceCat404) [reply modular_splurt/code/modules/mob/living/emotes.dm:1638]: Pfft! pretty sure that came from the *mrrp emote and I never took it off lol

File: Coder Build.bat
Comment (MosleyTheMalO) [inline Coder Build.bat:1]: please delete this file

**Pull request merged**